modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/LiferayExperienceCloud/components/SetupLXCModal/components/SetupLXCForm/components/SetupLXCPage/index.jsx

```diff
@@ -34,10 +34,60 @@ const SetupLiferayExperienceCloudPage = ({
 	values,
 }) => {
 	const [baseButtonDisabled, setBaseButtonDisabled] = useState(true);
-	const bannedDomains = useBannedDomains(
-		values?.lxc?.incidentManagementEmail,
-		500
-	);
+	const [
+		addHighPriorityContactList,
+		setAddHighPriorityContactList,
+	] = useState([]);
+	const [
+		removeHighPriorityContactList,
+		setRemoveHighPriorityContactList,
+	] = useState([]);
+	const [step, setStep] = useState(1);
+
+	const handlePreviousStep = () => {
+		setStep(step - 1);
+	};
+
+	const handleNextStep = () => {
+		setStep(step + 1);
+	};
+
+	const handleButtonClick = () => {
+		// eslint-disable-next-line no-unused-expressions
+		step === 1 ? handleOnLeftButtonClick() : handlePreviousStep();
+	};
```

Comment:
Try to use an 'enum' or a object to identify each step here and in the other steps for better organization if possible.